src/screens/TokenDetail.js

```diff
@@ -79,11 +79,9 @@ function TokenDetail() {
     }
     setToken(oldToken => {
       return {
-        token: {
-          ...oldToken,
-          uid: id,
-          meta: data,
-        },
+        ...oldToken,
+        uid: id,
```

Comment:
question(if-minor): Should we stop resetting the id?

Reply:
I believe we should but this screen should be redesigned. It is like this because there are two async requests and both of them set the uid. If this is the first response, then it must set the uid correctly.

I think the screen should be re-implemented in a way that it shows the screen after both requests finish, then we have the completed information and we don't need to set the id twice.

I prefer keeping like this and only fix the bug. What do you think?

Reply:
Makes sense